guides/common/modules/proc_creating-an-activation-key.adoc

```diff
@@ -44,7 +35,8 @@ endif::[]
 --content-view "_Stack_" \
 --organization "_My_Organization_"
 ----
-. Optional: For {RHEL} 8 hosts, enter the following command to configure the activation key with system purpose to set on hosts during registration to enhance subscriptions auto attachment.
+. Optional: Enter the following command to configure the activation key with system purpose attributes to set on hosts during registration.
+This helps determine which repositories are available on the host as well as with reporting in the Subscriptions service of the Red Hat Hybrid Cloud Console.
```

Comment:
```suggestion
This helps determine which repositories are available on the host.
It also helps with reporting in the Subscriptions service of the Red Hat Hybrid Cloud Console.
```

Reply:
I don't see this anymore in my text editor either.